client/src/elm/Translate.elm

```diff
@@ -13264,7 +13264,10 @@ translationSet trans =
         PrenatalActivityTitle activity ->
             case activity of
                 Backend.PrenatalActivity.Model.DangerSigns ->
-                    translationSet DangerSigns
```

Comment:
revert this

Reply:
Here on prenatal the Kirundi translation is different from the other one.

Reply:
Ok, add a comment about it next to this translation.